Extensions/DraggableBehavior/tests/draggableruntimebehavior.spec.js

```diff
@@ -96,6 +104,70 @@ describe('gdjs.DraggableRuntimeBehavior', function () {
       expect(object.getY()).to.be(700);
     });
 
+    it('can drag an object without collision mask check', function () {
+      object.setPosition(450, 500);
+      object.rotateTowardAngle(45, 0, runtimeScene);
+
+      // Should drag in bounding box but not in hitbox
+      runtimeScene.renderAndStep(1000 / 60);
+      runtimeGame.getInputManager().onMouseMove(450, 500);
+      runtimeGame
+        .getInputManager()
+        .onMouseButtonPressed(gdjs.InputManager.MOUSE_LEFT_BUTTON);
+      runtimeScene.renderAndStep(1000 / 60);
+      runtimeGame.getInputManager().onMouseMove(750, 600);
+      runtimeScene.renderAndStep(1000 / 60);
+      runtimeGame
+        .getInputManager()
+        .onMouseButtonReleased(gdjs.InputManager.MOUSE_LEFT_BUTTON);
+      runtimeScene.renderAndStep(1000 / 60);
+
+      expect(object.getX()).to.be(750);
+      expect(object.getY()).to.be(600);
+
+      object.rotateTowardAngle(0, 0, runtimeScene);
+    });
+
+    it('can drag an object with collision mask check', function () {
+      object2.setPosition(450, 500);
+      object2.rotateTowardAngle(45, 0, runtimeScene);
+
+      // Should not drag in bounding box but not in hitbox
```

Comment:
Could you rephrase these comments in the test?
It's a bit hard to understand the double negation "not drag in bounding box but not in hitbox".
For example, explain what is "in" (the dragged point is in the bounding box but the dragged point is also not in the hitbox).